Ventcamp_HTML/assets/js/ventcamp.js

```diff
@@ -1197,6 +1233,8 @@ Ventcamp = {
         this.gallerySliderInit();
 
         this.setEventHandlers();
+
+        this.videoPosition();

```

Comment:
To prevent the refresh issue you should listen on the window resize event, either through `window.onresize = function() {}` or jquery'wise `$(window).on('resize', function(){}`

Reply:
or maybe it's not a relevent issue. It's your call. I'll be happy to merge like it is. 👍

Reply:
but will it trigger on the initial page load?

Reply:
You can listen on both, but again. In real life situations, resize will almost never happen.